vmdk_plugin/Makefile

```diff
@@ -347,6 +347,11 @@ test-esx:
 testremote: test-esx test-vm
 test-all:  test e2e-test testremote
 
+# capture coverage of ESX Python code.
```

Comment:
minor: `capture coverage of ESX Python code` => `capture coverage of vDVS driver code`

Ref: https://i2.wp.com/www.virtuallyghetto.com/wp-content/uploads/2016/05/docker-volume-driver-for-vsphere-00.png